Mathlib/Algebra/Order/Monoid/Lemmas.lean

```diff
@@ -315,6 +315,27 @@ by simp_rw [min_le_iff, le_max_iff]; contrapose! h; exact mul_lt_mul_of_lt_of_lt
 #align min_le_max_of_add_le_add min_le_max_of_add_le_add
 #align min_le_max_of_mul_le_mul min_le_max_of_mul_le_mul
 
+end LinearOrder
+
+section LinearOrder
+variable [LinearOrder α] [CovariantClass α α HMul.hMul LE.le]
+  [CovariantClass α α (swap HMul.hMul) LE.le] {a b c d : α}
```

Comment:
Sorry @YaelDillies, I was wrong; I was thinking of #6777, but it's not actually merged after all.